src/FileFormats/MPS/MPS.jl

```diff
@@ -1319,7 +1319,7 @@ function parse_name_line(data::TempMPSModel, line::String)
     if m === nothing
         error("Malformed NAME line: ", line)
     end
-    data.name = strip(m[1])
+    data.name = strip(m[1]::AbstractString)
```

Comment:
There's actually a bunch of problems where we assume that the regex finds something.